keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2166,6 +2215,10 @@ k8s.io/kubelet/pkg/apis/dra gRPC interface. It was inspired by
 [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md),
 with “volume” replaced by “resource” and volume specific parts removed.
 
+Versions v1alpha4 and v1beta1 are supported by kubelet. Both are identical.
+DRA drivers should implement both because support for v1alpha4 might get
+removed.
```

Comment:
If you are curious about the technical aspect of implementing this, see the ["DRA kubelet: add v1beta1 gRPC API" commit](https://github.com/kubernetes/kubernetes/pull/127511).